gesturelibrary/src/double.c

```diff
@@ -51,13 +63,19 @@ static void update_double_taps(double_tap_t* double_tap, stroke_t* stroke, tap_t
                 double_tap->state = RECOGNIZER_STATE_COMPLETED;
                 break;
             }
+            if (on_double_tap) {
+                on_double_tap(double_tap);
+            }
```

Comment:
Since double_tap data is updated when the recognizer is in progress, in particular the `t` field, I think it would make sense to send the double tap event whether or not the tap state is completed.

In particular, consider moving the data updating BEFORE the state machine switch statement to facilitate this change.